packages/loaders/stories/1-Readme.stories.mdx

```diff
@@ -0,0 +1,6 @@
+import { Meta, Description } from '@storybook/addon-docs/blocks';
+import README from '../README.md';
```

Comment:
[nit] if we're going to show 3 loaders on the README, can we go ahead and show an example of all 5?

Reply:
Updated to now show all 5